src/cli-app/src/main/java/org/locationtech/geogig/cli/CLI.java

```diff
@@ -35,6 +40,38 @@ public static void main(String[] args) {
         final GeogigCLI cli = new GeogigCLI(consoleReader);
         cli.setRepositoryURI(repoURI);
 
+        GeoGIG geogig = cli.getGeogig();
+        boolean disableAnsi = false;
+        boolean closeIt = geogig == null;
+        if (closeIt) {
+            // we're not in a repository, need a geogig anyways to check the global config
+            geogig = cli.newGeoGIG(Hints.readOnly());
+            if (geogig.command(ConfigOp.class).setScope(ConfigScope.GLOBAL)
+                    .setAction(ConfigAction.CONFIG_GET).setName("ansi.enabled").toString()
+                    .equalsIgnoreCase("false")) {
+                disableAnsi = true;
+            } else if (geogig.command(ConfigOp.class).setScope(ConfigScope.GLOBAL)
+                    .setAction(ConfigAction.CONFIG_GET).setName("ansi.enabled").toString()
+                    .equalsIgnoreCase("true")) {
+                disableAnsi = false;
+            }
```

Comment:
shouldn't the ```geogig``` instance be closed here? (it checked for whether it had to be closed first)